src/components/FAQ/Faq.jsx

```diff
@@ -0,0 +1,70 @@
+import React, { useState } from "react";
+import useSWR from "swr";
+import { HeaderTitle } from "../../shared/Titles/HeaderTitle";
+import PreLoader from "../Loader/PreLoader";
+import { HiMinusCircle, HiPlusCircle } from "react-icons/hi";
+
+export const Faq = () => {
+  const fetcher = (url) => fetch(url).then((res) => res.json());
+  const { data, error } = useSWR("/api/getFaqData", fetcher);
+  const [selected, setSelected] = useState(null);
+
+  if (error) return <div>Failed to load</div>;
+
+  if (!data)
+    return (
+      <div>
+        <PreLoader />
+      </div>
+    );
+
+  const toggle = (i) => {
+    if (selected === i) {
+      return setSelected(null);
+    }
+    setSelected(i);
+  };
+
+  return (
+    <>
+      <div className="relative z-10 container px-4 mx-auto">
+        <div className="md:max-w-4xl mx-auto">
+          <HeaderTitle title={data?.headerTitle} key={data?.id} />
+          <div className="mb-11 flex flex-wrap mt-10">
+            {data?.content.map((item, i) => (
+              <div key={i} className="w-full p-1" onClick={() => toggle(i)}>
```

Comment:
check undefined ex: content?.map

Reply:
content defined in pages/api/FaqData.js

Reply:
as a practise when using map check undefined even the data const